docs/changelog.rst

```diff
@@ -11,6 +11,9 @@ for Python 3.6).
 
 New features:
 
+* Add the `substitute` plugin that is inspired to `rewrite` plugin. The `substitute` plugin does not modify the metadata
```

Comment:
Not sure about this wording, I think with the word inspired you would use `by`, not `to`

...is inspired by the `rewrite` plugin.

Reply:
....which is....

Reply:
Fixed